resources/options_lookup.tsv

```diff
@@ -40,11 +40,14 @@ Geometry Attic Type	None	ResStockArguments	geometry_attic_type=FlatRoof	geometry
 Geometry Attic Type	Vented Attic	ResStockArguments	geometry_attic_type=VentedAttic	geometry_roof_type=gable	geometry_roof_pitch=6:12																						
 Geometry Attic Type	Unvented Attic	ResStockArguments	geometry_attic_type=UnventedAttic	geometry_roof_type=gable	geometry_roof_pitch=6:12																						
 Geometry Attic Type	Finished Attic or Cathedral Ceilings	ResStockArguments	geometry_attic_type=ConditionedAttic	geometry_roof_type=gable	geometry_roof_pitch=6:12																						
-Geometry Building Type RECS	Single-Family Detached	ResStockArguments	geometry_unit_type=single-family detached	geometry_average_ceiling_height=8	geometry_unit_aspect_ratio=1.8																						
-Geometry Building Type RECS	Single-Family Attached	ResStockArguments	geometry_unit_type=single-family attached	geometry_average_ceiling_height=8	geometry_unit_aspect_ratio=0.5556																						
-Geometry Building Type RECS	Multi-Family with 2 - 4 Units	ResStockArguments	geometry_unit_type=apartment unit	geometry_average_ceiling_height=8	geometry_unit_aspect_ratio=0.5556																						
-Geometry Building Type RECS	Multi-Family with 5+ Units	ResStockArguments	geometry_unit_type=apartment unit	geometry_average_ceiling_height=8	geometry_unit_aspect_ratio=0.5556																						
-Geometry Building Type RECS	Mobile Home	ResStockArguments	geometry_unit_type=single-family detached	geometry_average_ceiling_height=8	geometry_unit_aspect_ratio=1.8																						
+Geometry Building Type RECS	Single-Family Detached	ResStockArguments	geometry_unit_type=single-family detached	geometry_unit_aspect_ratio=1.8																						
+Geometry Building Type RECS	Single-Family Attached	ResStockArguments	geometry_unit_type=single-family attached	geometry_unit_aspect_ratio=0.5556																						
+Geometry Building Type RECS	Multi-Family with 2 - 4 Units	ResStockArguments	geometry_unit_type=apartment unit	geometry_unit_aspect_ratio=0.5556																						
+Geometry Building Type RECS	Multi-Family with 5+ Units	ResStockArguments	geometry_unit_type=apartment unit	geometry_unit_aspect_ratio=0.5556																						
+Geometry Building Type RECS	Mobile Home	ResStockArguments	geometry_unit_type=single-family detached	geometry_unit_aspect_ratio=1.8																						
+Geometry Ceiling Height	8 ft	ResStockArguments	geometry_average_ceiling_height=8					
+Geometry Ceiling Height	9 ft	ResStockArguments	geometry_average_ceiling_height=9					
+Geometry Ceiling Height	10 ft	ResStockArguments	geometry_average_ceiling_height=10					

```

Comment:
same question about `Cathedral Ceiling`

Reply:
Mapping it to 10 ft, on the em side.

https://github.com/T110-nrgx/energy-modeling/pull/171#pullrequestreview-1684262924